File: COMP9041/lab07/count_word.py
#!/usr/bin/python

import sys
import re

count = []
key = (sys.argv[1]).lower()
for line in sys.stdin:
	pattern = re.compile(r"\b%s\b" %key, re.IGNORECASE)
	count += re.findall(pattern, line)
print (key + " occurred " + str(len(count)) + " times")

File: COMP9041/lab07/count_word.pl
#!/usr/bin/perl -w

$count = 0;
$key = lc $ARGV[0];
while (<STDIN>) {
        @words = split /[^A-Za-z]/, $_;
        foreach(@words) {
                if (lc $_ eq $key) {
                        $count ++;
                }
        }
}

print "$key occurred $count times\n";


